packages/nextjs/components/Header.tsx

```diff
@@ -54,8 +68,64 @@ export default function Header() {
               Example UI
             </NavLink>
           </li>
+          <li>
+            <NavLink href="/ninja">
+              <BeakerIcon className="h-4 w-4" />
+              Ninja
+            </NavLink>
+          </li>
         </ul>
       </div>
+      <div className="navbar bg-base-100">
+        <div className="navbar-start">
+          <div className="dropdown sm:hidden">
+            <label
+              tabIndex={0}
+              className="btn btn-ghost ml-1 hover:bg-secondary"
+              onMouseDown={e => {
+                checkAndCloseDropDown(e);
+              }}
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                className="h-5 w-5"
+                fill="none"
+                viewBox="0 0 24 24"
+                stroke="currentColor"
+              >
+                <path strokeLinecap="round" strokeLinejoin="round" strokeWidth="2" d="M4 6h16M4 12h8m-8 6h16" />
+              </svg>
+            </label>
+            <ul
+              tabIndex={0}
+              className="menu menu-compact dropdown-content mt-3 p-2 shadow bg-base-100 rounded-box w-52"
+            >
+              <li>
+                <NavLink href="/">
+                  <HomeModernIcon className="h-4 w-4" />
+                  Home
+                </NavLink>
+                <NavLink href="/debug">
+                  <BugAntIcon className="h-4 w-4" />
+                  Debug Contracts
+                </NavLink>
+                <NavLink href="/example-ui">
+                  <SparklesIcon className="h-4 w-4" />
+                  Example UI
+                </NavLink>
+                <NavLink href="/ninja">
+                  <BeakerIcon className="h-4 w-4" />
+                  Ninja
+                </NavLink>
```

Comment:
You are repeating the same links here, which it's not good DX (having to change in two places).

If it's the same content, we could move it to a component above. If we need a different markup, we could have an array with the data 

```js
const menuItems = [
 {
  label: "Home",
  link: "/"
}
//...
  
];
```